starter-code/styles/base.css

```diff
@@ -46,3 +46,6 @@ a:visited {
 /*FORM STYLES
 =======================================*/
 /*TODO: Add the CSS needed to style the form*/
+label {
+  display:block;
```

Comment:
Need to do some bare bones styling to the form. At least give it some spacing and make it more readable.